WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostRenderer.java

```diff
@@ -277,6 +281,23 @@ private String getPostContent() {
         return content;
     }
 
+    /*
+     * Strips inline styles from post content
+     */
+    private String removeInlineStyles(String content) {
+        if (content.length() == 0) {
+            return content;
+        }
+
+        try {
+            Document doc = Jsoup.parse(content);
+            doc.getAllElements().removeAttr("style");
+            return doc.select("body").toString();
```

Comment:
Nice solution!

I noticed that the returned HTML included the outer `body` tag, which appears to have been generated by jsoup since it wasn't included in the original `content` string. Do you think we should just return everything inside `body`?

[input.txt](https://github.com/wordpress-mobile/WordPress-Android/files/7549196/input.txt)
[output.txt](https://github.com/wordpress-mobile/WordPress-Android/files/7549197/output.txt)

Reply:
Nice catch, 60a5853 changes that.